index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/styles.css" />
-    <title>Portfolio - Setup and Mobile First</title>
+    <title>Portfolio - Desktop version</title>
```

Comment:
// Well! 👍 I noticed that you have added a page title using the `<title>` tag in the `<head>` of the page, which is a good practice. However, upon reviewing the title, I suggest making some changes. The page title should adequately and briefly describe the content of the page to provide clarity to the users and search engines.

For example:
1. "Juan Carlos Portfolio"
2. "Juan Carlos - Full-Stack Web Developer"

By updating the page title to accurately reflect the content of your portfolio, you will improve the accessibility and SEO optimization of your website. Keep up the great work! 🌟